com.liferay.blade.cli/src/com/liferay/blade/cli/ServerCommand.java

```diff
@@ -80,6 +80,12 @@ private void commandServer(
 			String cmd, File dir, String serverType, ServerOptions options)
 		throws Exception {
 
+		if (!dir.exists() || dir.listFiles() == null ) {
+			_blade.error(serverType + " bundles do not exist in Liferay Workspace, 'gradlew initBundle' command may be helpful.");
```

Comment:
Change the message to say 
`" bundles folder does not exist in Liferay workspace, execute 'gradle initBundle' in order to create it'